### PR TITLE
Add team and royalties guides

### PR DESCRIPTION
**Pull request by antho1404**
Add new guide to set teams and update royalties

### CODE REVIEW

File: docs/pages/guides/inviting-member.mdx [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+| Role             | Permissions                                           | Best For                              |
+| ---------------- | ----------------------------------------------------- | ------------------------------------- |
+| Financial        | Access trade activity and offers on the platform      | Financial oversight and management    |
+| Developer        | Create webhooks and custom plugins                    | Technical development and integration |
+| Management       | Access and edit collections, drops, assets, and users | Overall platform management           |
+| Settings         | Update the settings of the platform                   | Platform configuration                |
+| Collection Owner | Set royalties for the collection they own             | Managing individual collections       |
+
+![Team Member Roles](/mnt/data/Screenshot 2024-02-07 at 14.58.52.png)
```
Comment (EmmanuelDrouin) [inline docs/pages/guides/inviting-member.mdx]: screenshot not displayed on the doc

File: docs/pages/guides/inviting-member.mdx [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+## Inviting a Team Member
```
Comment (EmmanuelDrouin) [inline docs/pages/guides/inviting-member.mdx]:
```suggestion
## Steps to Invite a Team Member
```

File: docs/pages/guides/inviting-member.mdx [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+
+<Callout type="warning">
```
Comment (EmmanuelDrouin) [inline docs/pages/guides/inviting-member.mdx]: Replace warning by info box, as this content is purely informative

File: docs/pages/guides/inviting-member.mdx [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+  security and integrity of the platform.
+</Callout>
+
+To invite a new team member:
+
+<Steps>
+
+### Go to Team Management
+
+Navigate to the 'Team' section on the Liteflow dashboard.
```
Comment (EmmanuelDrouin) [inline docs/pages/guides/inviting-member.mdx]:
```suggestion
Navigate to the '[Team](https://dashboard.liteflow.com/settings/team)' section on the Liteflow dashboard.
```

File: docs/pages/guides/inviting-member.mdx [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+
+### Enter Details
+
+In the 'Add an admin' dialog, enter the wallet address and email of the team member you are inviting.
+
+### Assign Roles
+
+Check the appropriate roles based on the permissions you wish to grant to the new member.
+
+### Send Invitation
```
Comment (EmmanuelDrouin) [inline docs/pages/guides/inviting-member.mdx:58]: We are missing the last part that is: from the user invited what do they receive and how?

From the flow I understand what I'm doing as a platform manager by inviting someone, but currently I have no idea how the invitation looks like, how it will be send, etc

Missing end flow (cf comment from Hodooi on Royalties that is similar)

File: docs/pages/guides/setting-royalties.mdx [hunk 6]
```diff
@@ -0,0 +1,54 @@
+---
+title: 'Setting Royalties'
```
Comment (EmmanuelDrouin) [inline docs/pages/guides/setting-royalties.mdx]:
```suggestion
title: 'Setting up Royalties'
```

File: docs/pages/guides/setting-royalties.mdx [hunk 7]
```diff
@@ -0,0 +1,54 @@
...
+---
+
+import { Steps, Callout } from 'nextra-theme-docs'
+
+# Setting Royalties for Collections
```
Comment (EmmanuelDrouin) [inline docs/pages/guides/setting-royalties.mdx]:
```suggestion
# Setting up Royalties for Collections
```

File: docs/pages/guides/setting-royalties.mdx [hunk 8]
```diff
@@ -0,0 +1,54 @@
...
+
+This guide provides instructions for platform owners and collection owners on how to set royalties for a collection using the Liteflow dashboard.
```
Comment (EmmanuelDrouin) [inline docs/pages/guides/setting-royalties.mdx]:
```suggestion
This guide provides instructions for platform and collection owners on how to set royalties for a collection using the Liteflow dashboard.
```

File: docs/pages/guides/setting-royalties.mdx [hunk 9]
```diff
@@ -0,0 +1,54 @@
...
+
```
Comment (EmmanuelDrouin) [inline docs/pages/guides/setting-royalties.mdx:8]:
```suggestion
## Setting up Royalties
```

File: docs/pages/guides/setting-royalties.mdx [hunk 10]
```diff
@@ -0,0 +1,54 @@
...
+<Callout type="info">
+  If you are a platform owner and do not own the collection for which you wish
+  to set royalties, you will need to [invite the collection
+  owner](/guides/inviting-member#inviting-a-team-member) with the role
+  ["Collection Owner"](/guides/inviting-member#team-roles-and-permissions) to
+  your platform.
+</Callout>
+
```
Comment (EmmanuelDrouin) [inline docs/pages/guides/setting-royalties.mdx]:
```suggestion
To set royalties for a collection:
```

File: docs/pages/guides/setting-royalties.mdx [hunk 11]
```diff
@@ -0,0 +1,54 @@
...
+<Steps>
+
+### Log In to the Platform
+
+Access the Liteflow dashboard and navigate to the ["Collections" section](https://dashboard.liteflow.com/collections) to view all collections you own.
```
Comment (EmmanuelDrouin) [inline docs/pages/guides/setting-royalties.mdx]:
```suggestion
Access the Liteflow dashboard and navigate to the ["Collections" section](https://dashboard.liteflow.com/collections) to view all collections available on your platform.
```

**Comment by EmmanuelDrouin** [comment]
@antho1404 I have edited the content directly in some parts to improve and bring consistency. Good guides!

One thing that is missing from Hodooi's feedback is the end flow from the point of view of the user who received the invitation, on the invite team members.
That was the specific point he mentioned to me: how does the invitation looks like, what do they receive, through which channel, etc.
- Basically anything that can make him understand and visualize what's happening after he clicks the "send invitation"
  - This does not only apply to collection's owner, but to everyone we invite as we do not explain the end of the flow regarding invitation received

File: docs/pages/guides/inviting-member.mdx (hunk 5 above)
Comment (antho1404) [reply docs/pages/guides/inviting-member.mdx:58]: I added a section in the last step [9575fe9](https://github.com/liteflow-labs/liteflow-js/pull/223/commits/9575fe9a902dae797cf5614fda0b19da0a30f3a5)

**Comment by antho1404** [comment]
I added the changes, merging this PR but feel free to review and create an issue if anything is wrong
